perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTestAbstract.java

```diff
@@ -7282,6 +7282,245 @@ public void getRequiredHostAttributesFromOneServiceWhenHostNotExists() throws Ex
 
 // already tested in lower layer - package: cz.metacentrum.perun.core.impl.modules.attributes
 
+	@Test
+	public void testCheckAttributeSyntaxVo() throws Exception {
+		attributesManagerBl = getTargetObject(perun.getAttributesManagerBl());
+		Attribute attribute = setAttributeInNamespace(AttributesManager.NS_VO_ATTR_DEF);
+
+		attributesManagerBl.checkAttributeSyntax(sess, vo1, attribute);
```

Comment:
Please call checkAttributeSyntax from attributesManager instead of attributesManagerBl if it is possible (in all tests). We need to check if the code in the Entry layer is working correctly too.

Reply:
Okay, it should use the correct manager now, thank you.